src/plan/global.rs

```diff
@@ -710,12 +710,14 @@ impl<VM: VMBinding> BasePlan<VM> {
     pub fn set_gc_status(&self, s: GcStatus) {
         let mut gc_status = self.gc_status.lock().unwrap();
         if *gc_status == GcStatus::NotInGC {
+            probe!(mmtk, gc_start);
```

Comment:
Why not put this to the controller loop which is more reliable than `GcStatus`? 
https://github.com/mmtk/mmtk-core/blob/5d22c2b3ca6c0f3393f9c8cbd8b47a28868a48cc/src/scheduler/controller.rs#L54

We set `GcStatus::GcProper` in these places, and we have executed some packets before that.
https://github.com/mmtk/mmtk-core/blob/5d22c2b3ca6c0f3393f9c8cbd8b47a28868a48cc/src/scheduler/gc_work.rs#L452
https://github.com/mmtk/mmtk-core/blob/5d22c2b3ca6c0f3393f9c8cbd8b47a28868a48cc/src/scheduler/gc_work.rs#L475

Reply:
Fixed